lib/crowdnode.js

```diff
@@ -74,23 +77,25 @@ CrowdNode.responses = {
  * @param {Object} opts
  * @param {String} opts.baseUrl
  * @param {String} opts.insightBaseUrl
+ * @param {String} opts.knowledgeBaseUrl
  */
-CrowdNode.init = async function ({ baseUrl, insightBaseUrl }) {
+CrowdNode.init = async function ({ baseUrl, insightBaseUrl, knowledgeBaseUrl }) {
   // TODO use API
   // See https://github.com/dashhive/crowdnode.js/issues/3
 
   CrowdNode._baseUrl = baseUrl;
+  CrowdNode._knowledgeBaseUrl = knowledgeBaseUrl || CrowdNode.main.knowledge;
 
   //hotwallet in Mainnet is XjbaGWaGnvEtuQAUoBgDxJWe8ZNv45upG2
   CrowdNode.main.hotwallet = await request({
     // TODO https://app.crowdnode.io/odata/apifundings/HotWallet
```

Comment:
TODO => TODONE, please

Reply:
For documentation purposes and per our other conversation, Those TODO's aren't actually resolved, this just allows bypassing a CORS issue in the browser in vite.